[doc] Fix inconsistencies with torch.linalg.cond doc

heitorschueroff opened the pull request:
fixes https://github.com/pytorch/pytorch/issues/51643

Stack from [ghstack](https://github.com/ezyang/ghstack):
* #51672 [doc] Fix inconsistencies with torch.linalg.inv and deprecate torch.inverse
* #51671 [doc] Fix inconsistencies with linalg.pinv docs and deprecate pinverse
* #51660 doc] Fix inconsistencies with torch.linalg.matrix_rank doc
* #51659 [doc] Fix inconsistencies with torch.linalg.eigvalsh
* #51658 [doc] Fix inconsistencies with torch.linalg.eigh
* #51651 [doc] Fix inconsistencies with torch.linalg.det docs
* **#51641 [doc] Fix inconsistencies with torch.linalg.cond doc**

Differential Revision: [D26234104](https://our.internmc.facebook.com/intern/diff/D26234104)

heitorschueroff committed:
[doc] Fix inconsistencies with torch.linalg.cond doc

[ghstack-poisoned]

IvanYashchuk commented on torch/linalg/__init__.py:
```diff
@@ -602,34 +602,32 @@
-
 Args:
     input (Tensor): the input matrix of size :math:`(m, n)` or the batch of matrices of size :math:`(*, m, n)`
-                    where `*` is one or more batch dimensions.
+                    where :math:`*` is one or more batch dimensions.
```
I suggest using just \` \`, without :math:. Inline math looks odd to me, it is more useful for standalone equations.
With just \` \` it would look like this
![image](https://user-images.githubusercontent.com/19621411/106795540-df0e0c00-6662-11eb-8983-ac118a187e92.png)
This example is taken from current `torch.svd` docs.

IvanYashchuk reviewed (commented):
Here is a link to the rendered version https://10638833-65600975-gh.circle-artifacts.com/0/docs/linalg.html#torch.linalg.cond

heitorschueroff commented on torch/linalg/__init__.py:
```diff
@@ -602,34 +602,32 @@
+
 For norms ``p = {'fro', 'nuc', inf, -inf, 1, -1}`` this is defined as the matrix norm of :attr:`input`
-times the matrix norm of the inverse of :attr:`input`. And for norms ``p = {None, 2, -2}`` this is defined as
-the ratio between the largest and smallest singular values.
+times the matrix norm of the inverse of :attr:`input` computed using :func:`torch.linalg.norm`. While
+for norms ``p = {None, 2, -2}`` this is defined as the ratio between the largest and smallest singular
```
I'll try writing these in LaTex see if it looks good.

heitorschueroff committed:
Update on "[doc] Fix inconsistencies with torch.linalg.cond doc"




[ghstack-poisoned]

heitorschueroff committed:
Update on "[doc] Fix inconsistencies with torch.linalg.cond doc"


Differential Revision: [D26234104](https://our.internmc.facebook.com/intern/diff/D26234104)

[ghstack-poisoned]

heitorschueroff committed:
Update on "[doc] Fix inconsistencies with torch.linalg.cond doc"


Differential Revision: [D26234104](https://our.internmc.facebook.com/intern/diff/D26234104)

[ghstack-poisoned]

heitorschueroff committed:
Update on "[doc] Fix inconsistencies with torch.linalg.cond doc"


Differential Revision: [D26234104](https://our.internmc.facebook.com/intern/diff/D26234104)

[ghstack-poisoned]

heitorschueroff committed:
Update on "[doc] Fix inconsistencies with torch.linalg.cond doc"


Differential Revision: [D26234104](https://our.internmc.facebook.com/intern/diff/D26234104)

[ghstack-poisoned]

heitorschueroff commented on torch/linalg/__init__.py:
```diff
@@ -602,34 +602,32 @@
-                    where `*` is one or more batch dimensions.
+                    where :math:`*` is one or more batch dimensions.
 
     p (int, float, inf, -inf, 'fro', 'nuc', optional): the type of the matrix norm to use in the computations.
```
I've improved the rendering. It's still not perfect as it breaks into a new line but the same happens to linalg.norm. Let me know what you think.

heitorschueroff committed:
Update on "[doc] Fix inconsistencies with torch.linalg.cond doc"

fixes https://github.com/pytorch/pytorch/issues/51643


Differential Revision: [D26234104](https://our.internmc.facebook.com/intern/diff/D26234104)

[ghstack-poisoned]
